PortfolioStyle.css

```diff
@@ -0,0 +1,219 @@
+
+/*Font*/
+
+@import url('https://fonts.googleapis.com/css2?family=Josefin+Sans:ital,wght@0,100;0,200;0,400;1,100;1,200&display=swap');
+	
+/*General CSS - normalize file contains most of it */
+	
+* {
+	box-sizing: border-box;
+}  
+	
+/* Using rems - set up to be 10px = 1 rem*/
+
+body {
+	font-size: 62.5%;
+	font-family: "Josefin Sans";
+	font-weight: 100;
+	background-color: #FCF7F8;
+}
+/* Colours
+
+Background: #FCF7F8 (Pinky White)
+Borders: #CED3DC (Grey)
+Black ish: #000500
+Hover color: #ABA9C3 (Lilac)
+Accents: #88292F (Red)
+
+*/
+
+
+
+/*Text*/
+
+h1 {
+	font-size: 5rem;
+}
+
+h2 {
+	font-size: 2rem;
+	text-align: center;
+	font-weight: 200;
+	padding: 20px 0 0 0;
+}
+
+h3 {
+	font-size: 1.5rem;
+}
+
+h4 {
+	font-size: 1rem;
+	font-weight: 200;
+}
+h6 {
+	font-size: 0.9rem;
+	padding: 20px;
+	font-weight: 100;
+}
+
+p {
+	font-size: 1.2rem;
+	text-align: center;
+}
+
+
+/*Nav Bar*/
+
+nav {
+	overflow: hidden;
+	position:fixed;
+	top: 0;
+	width: 100%;
+	margin: 0;
+	font-size: 1.5rem;
+	background-color: #FCF7F8;
+}
+
+li {
+	float: left;	
```

Comment:
You can achieve the same affect with flexbox instead of float, flexbox also gives you responsiveness in container size.

Reply:
Done!